### PR TITLE
[RW-693] Adding ability to specify table=<table class> on TableQuery

### PR DESCRIPTION
**Pull request by danqrodney**

### CODE REVIEW

File: py/aou_workbench_client/swagger_client/models/table_query.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def __init__(self, table_name=None, columns=None, filters=None, order_by=None):
         self._order_by = None
         self.discriminator = None
 
-        self.table_name = table_name
+        self.table_name = table_name or (table.table_name if table else None)
```
Comment (fonnesbeck) [inline py/aou_workbench_client/swagger_client/models/table_query.py]: Clearer just to say `table.table_name if table else table_name`?

File: py/test/cohorts_api_test.py [hunk 2]
```diff
@@ -18,6 +20,22 @@ def test_materialize_cohort_page(self):
       self.assertEqual(10, len(response_2.results))
       self.assertIsNotNone(response_2.next_page_token)
       self.assertNotEqual(response_2.results, response.results)
+    
+    def test_materialize_cohort_table_query(self):
+      table_query = TableQuery(table=Person, columns=[Person.person_id])
```
Comment (fonnesbeck) [inline py/test/cohorts_api_test.py]: Should be 4 spaces on indent

File: py/aou_workbench_client/swagger_client/models/table_query.py (hunk 1 above)
Comment (danqrodney) [reply py/aou_workbench_client/swagger_client/models/table_query.py]: Done.

File: py/test/cohorts_api_test.py (hunk 2 above)
Comment (danqrodney) [reply py/test/cohorts_api_test.py]: Done.

**Comment by fonnesbeck** [comment]
LGTM

**Pull request merged**